std_msgs/Header header
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
geometry_msgs/Vector3 linear_velocity
geometry_msgs/Vector3 angular_velocity
float64 pitch
float64 roll
float64 yaw
float64 q1
float64 q2
float64 q3
float64 q4
float64 m1
float64 m2
float64 m3
float64 m4
int64 vehicleID
float64 posx
float64 posy
float64 posz
float64 attyaw
float64 attpitch
float64 attroll
float64 attq0
float64 attq1
float64 attq2
float64 attq3
